lib/string_transformer.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module StringTransformer
```

Comment:
This name feels _much broader_ than this is in practice? I also think its actually more app-specific than a conventional lib file (lib should be things you can copy-paste into most projects, whereas logic around friendly ids and handles is pretty _domain specific_ for us?)

Reply:
We should also make sure we use this class throughout wherever we're doing stuff in this space of handle/friendly id generation